### PR TITLE
Added changes to steeplechase

### PR DESCRIPTION
**Pull request by RitheshShenthar**
1. Added change of path to html-manifest in webrtc tests
2. Added sunny day details into job template. (Corresponding global
   variables were hand configured on Jenkins machine)

### CODE REVIEW

File: webrtc-job-template.xml [hunk 1]
```diff
@@ -130,7 +130,7 @@ if [ "$PACKAGE1" != "$PACKAGE2" ] ; then
...
+python `pwd`/mozplatformqa-jenkins/jenkinsherder.py --steeplechase `pwd`/steeplechase/steeplechase/runsteeplechase.py --package $PACKAGE1 --package2 $PACKAGE2 --save-logs-to `pwd`/logs --specialpowers-path /home/mozilla/firefoxes/XXXTESTSRELEASEXXX/linux64/tests/steeplechase/specialpowers --prefs-file /home/mozilla/firefoxes/XXXTESTSRELEASEXXX/linux64/tests/steeplechase/prefs_general.js --signalling-server $XXXSIGSERVERXXX --html-manifest /home/mozilla/firefoxes/XXXTESTSRELEASEXXX/linux64/tests/dom/media/tests/mochitest/steeplechase.ini --host1 $MACHINE1:20701 --arch1 $ARCH1 --host2 $MACHINE2:20701 --arch2 $ARCH2 --os1 $OS1 --os2 $OS2 --treeherder-url $TREEHERDER_URL --treeherder-credentials-path $TREEHERDER_CREDENTIALS
```
Comment (sydvicious) [inline webrtc-job-template.xml:133]: This change looks good.

**Comment by sydvicious** [comment]
@RitheshShenthar: It's hard to tell what changed in the .csv file since the line endings changed. Could you let me know the line number that you actually changed?

**Comment by RitheshShenthar** [comment]
in the CSV look for the Row 27 with "sunnyday" details

**Comment by RitheshShenthar** [comment]
Thanks Syd!

**Pull request merged**